src/CMakeLists.txt

```diff
@@ -75,12 +75,14 @@ endif(DISABLE_FMAD)
 
 
 include_directories(${amrex_INCLUDE_DIRS_RET})
+include_directories(${Microphysics_INCLUDE_DIRS_RET})
 include_directories(${fmt_INCLUDE_DIRS_RET})
 include_directories(${HDF5_INCLUDE_DIRS_RET})
 include_directories(${Boost_INCLUDE_DIRS_RET})
 include_directories(${CMAKE_SOURCE_DIR}/src)
 
 link_libraries(AMReX::amrex)
+link_libraries(Microphysics::Microphysics)
```

Comment:
I think we can't link to Microphysics as a library for the reason that Mike mentioned -- different problems might want to use a different network, and we don't want to build all of the networks for every problem. So we just need to add the CMake variable listing the source files to `add_executable` inside `PrimordialChem/CMakeLists.txt`.

Reply:
Something about this line is also causing all of the tests to fail. Not sure why that is, but it should go away if you remove it.

Reply:
I removed the `link_libraries` line, but I do keep the `include_directories` line for microphysics?